Marathi/Shivaji-Maharaj/राजे-शिवाजी-महाराज.md

```diff
@@ -0,0 +1,24 @@
+छत्रपती शिवाजीराजे भोसले (१९ फेब्रुवारी १६३० ते ३ एप्रिल १६८०) हे इ.स. १८१८ पर्यंत टिकलेल्या आणि आपल्या 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.